src/lib/coda_lib/coda_lib.ml

```diff
@@ -188,7 +188,17 @@ let best_protocol_state = compose_of_option best_protocol_state_opt
 
 let best_ledger = compose_of_option best_ledger_opt
 
-let get_ledger t staged_ledger_hash =
+let get_ledger t staged_ledger_hash_opt =
+  let staged_ledger_hash =
+    Option.value staged_ledger_hash_opt
+      ~default:
+        ( match best_staged_ledger t with
+        | `Active staged_ledger ->
+            staged_ledger |> Staged_ledger.hash
+        | `Bootstrapping ->
+            failwith
+              "get_ledger: can't get staged ledger hash while bootstrapping" )
```

Comment:
we should never be able to crash the daemon triggered via an RPC call -- can we put this back to an `Or_error`?

Reply:
I'll try ...